plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/ThemeConfig.java

```diff
@@ -44,8 +44,17 @@
      * Sets the CSS relative paths of this theme configuration.
      *
      * @param css CSS relative paths to be set
+     * @throws IllegalArgumentException if a CSS file relative path starts with a '/'
      */
     public void setCss(List<String> css) {
+        if (css != null) {
+            for (String relativePath : css) {
+                if (relativePath.charAt(0) == '/') {
+                    throw new IllegalArgumentException(
+                            "CSS file relative path cannot start with a '/'. '" + relativePath + "' is incorrect.");
```

Comment:
I think this way looks better as the whole message is in a one line, which improves the readability.